### ISSUE TITLE
Linespacing in `pack_rows` not adjustable and group label placing in longtable error

### ISSUE THREAD

**Issue by jono3030** [haozhu233/kableExtra#480]
First of all, thanks for the great package!

Three separate issues:

1) Could you add a way to control `linesep` for grouped rows? At the moment, `pack_rows` automatically overrides `linesep` it seems. I would need more space between rows that are grouped for the table I'm building as it has a lot of text. Perhaps a `linesep` option within `pack_rows`?

2) Group labels sometimes get placed as the last row in a longtable instead of as the first row on the next page of the longtable. Is there any way to control this behavior ie by forcing the group label to the next page? Also, when using `hline_before` on a row that is the first row on a new page, the last hline on the bottom of the previous page is double. If I remove `hline_before` the distance between the group label and the top line is different than for the group labels with `hline_before`.

3) Is there a way to make the repeated headers bold as well?

```{r}
library(dplyr)
library(knitr)
library(kableExtra)

mtcars <- rbind(mtcars, mtcars, mtcars, mtcars)

mtcars %>%
	select(mpg, cyl, disp, hp, drat) %>%
        kable(.,
	  "latex",
          booktabs = T,
          longtable = T,
          caption="Test table",
          linesep = "\\addlinespace",
          escape = F) %>%
	  kable_styling(latex_options = c("repeat_header", "HOLD_position"),
				position = "center",
				full_width = T,
				font_size = 8,
				repeat_header_continued = T) %>%
          row_spec(0, bold = T) %>%
          column_spec(1, width = "20em") %>%
          pack_rows("Category 1", 3, 5,
                    latex_align = "c",
                    indent = F,
                    hline_before = T,
                    hline_after = T,
                    latex_gap_space = "0em") %>%
          pack_rows("Category 2", 6, 15,
                    latex_align = "c",
                    indent = F,
                    hline_before = T,
                    hline_after = T,
                    latex_gap_space = "0em") %>%
          pack_rows("Category 3", 46, 65,
                    latex_align = "c",
                    indent = F,
                    hline_before = T,
                    hline_after = T,
                    latex_gap_space = "0em") %>%
          pack_rows("Category 4", 103, 110,
                    latex_align = "c",
                    indent = F,
                    hline_before = T,
                    hline_after = T,
                    latex_gap_space = "0em")
```

Image of the issues:

![issues](https://user-images.githubusercontent.com/43489273/87217101-e57a1680-c313-11ea-833c-ab423c54bf79.png)

I couldn't replicate the issue where the group label is the last row on the wrong page but I found an image of a table where that happened. It also illustrates the issue with there not being enough line space between rows:

![issues2](https://user-images.githubusercontent.com/43489273/87217316-d1cfaf80-c315-11ea-943a-43a4787c0d7e.png)

**Comment by haozhu233** [comment]
@jono3030 Thanks for this issue. The only thing I can fix at this moment is item #3. For item #1, I found it's kind of difficult to think of a nice API. For item #2, I think this is more like a latex issue. As far as I know, it's not possible but if you know how to do that, I can try to integrate that into this pacakge. For item #3, it's fixed now.

**Comment by jono3030** [comment]
@haozhu233 Thanks a lot for looking at this and apologies for the late reply. I'll have a look at how Latex is handling item #2 behind the scenes and get back to you if I'm able to figure it out.

Thanks again for the great package!
